cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxMusic.java

```diff
@@ -29,9 +29,6 @@ of this software and associated documentation files (the "Software"), to deal
 import android.content.res.AssetFileDescriptor;
 import android.media.MediaPlayer;
 import android.util.Log;
-import android.content.res.AssetFileDescriptor;
-import com.android.vending.expansion.zipfile.APKExpansionSupport;
-import com.android.vending.expansion.zipfile.ZipResourceFile;
```

Comment:
Why remove these codes, it is needed for OBB. Please refer to #15515 for detail information.